Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'sidekiq' # background processing for Ruby
 gem 'bootsnap', require: false # Reduces boot times through caching; required in config/boot.rb
 gem 'i18n-js', '3.5.1' # A library to provide the I18n translations on the Javascript
 gem 'fast_jsonapi' # A lightning fast JSON:API serializer for Ruby Objects
+gem 'graphql' # Support GraphQL
```

Comment:
How about changing the description to "Ruby implementation of GraphQL" as usually, we rely on the project description. Minot thing though.